tools/auto_analyse_raw_data.py

```diff
@@ -415,20 +423,30 @@ def decode_data(message, defines, code, name="", output=sys.stdout):
       "  for (uint16_t r = 0; r <= repeat; r++) {",
       "    uint64_t send_data = data;"])
   code["send64+"].extend([
+      "/// Alternative >64bit function to send %s messages" % def_name.upper(),
```

Comment:
As this is a comment line more for the user running the prog, lets make it a non-doxygen comment.
i.e. just `//`